conceptnet5/vectors/formats.py

```diff
@@ -204,6 +206,10 @@ def load_word2vec_bin(filename, nrows):
             idx = len(label_list)
             arr[idx] = vec
             label_list.append(label)
+
+    if len(label_list) < nrows:
```

Comment:
Nitpick: here the list of labels is called `label_list`, while in `load_glove` it's `labels`. In other methods there is a mismatch as well: in `load_polyglot` it's `label_list`, while in 'load_labels_and_npy' it's `labels` again.